### DISCUSSION TITLE
Stuck with DDIM sampling instead of Euler a

### DISCUSSION THREAD

**Discussion by TexNorwegian** [glucauze/sd-webui-faceswaplab#150] [Q&A]
Everything was going fine this morning while doing some image generation. When I got home, I finished what I was doing and FaceSwapLabs was extremely slow generating. I restarted the A1111 thinking something was wrong. Nope.

Looking at the log while I was generating a simple 512x768 image showed _Running DDIM Sampling with 100 timesteps_ even though I was using _Euler a_ at the time. It took  1 min. 1.0 seconds for that.

```
LatentDiffusionV1: Running in eps-prediction mode
Keeping EMAs of 308.
Applying attention optimization: sdp... done.
Downsampling from [512, 768] to [128, 192]
Plotting: Switched to EMA weights
Sampling with eta = 1.0; steps: 100
Data shape for DDIM sampling is (1, 3, 192, 128), eta 1.0
Running DDIM Sampling with 100 timesteps
DDIM Sampler: 100%|██████████████████████████████████████████████████████████████████| 100/100 [00:15<00:00,  6.52it/s]
Plotting: Restored training weights
2023-11-06 18:22:44,884 - FaceSwapLab - INFO - Restore face with CodeFormer
2023-11-06 18:22:45,903 - FaceSwapLab - INFO - Add swp image to processed
Total progress: 100%|██████████████████████████████████████████████████████████████████| 35/35 [01:00<00:00,  1.72s/it]
```

I changed the sampler to DPM++ 2M SDE Karras and it's the same, DDIM doing the sampling. As such, my outputs look horrible.

I changed nothing in my FSL settings from this morning to this afternoon.

I have isolated it to FSL because this doesn't happen when not using FSL.

Do I need to do a reinstall of FSL?

**Comment by TexNorwegian** [comment]
After further investigating and advice, the DDIM thingy is supposed to be doing what it does regardless of my settings.

But...... my FSL extension is still continuously putting out horrid images. Again, Monday morning things were fine. Monday afternoon, everything went to hell as far is images go. They're horrible. I haven't changed a thing in any settings.

Reinstall FSL?

**Comment by TexNorwegian** [comment]
Discard this whole thread. I found my problem. I had edited my webui-user bat file and it messed things up big time.

**Reply by antorio** [reply]
@TexNorwegian What did you had edited? I think I have same problem, output faces gone terrible today, but not yesterday

**Reply by TexNorwegian** [reply]
I can't remember now what I had added to my webui-user .bat, but it was a friend of mine who recommended what to add to it. It was a last-ditch effort on my part, thinking it was my problem. After I returned my .bat file back the original state, things began to function as before. Posting what I have right now in my webui-user .bat file below.

```
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS= --medvram --no-half-vae --skip-install

call webui.bat --medvram --opt-split-attention
```

Looking back a minute ago, my buddy had said to add _--opt-split-attention_ to the _set COMMANDLINE_ARGS_ line.

When I removed it and put it back to _call webui.bat_, things on FSL began to function correctly again.
